site/docs/concepts/taxonomy.md

```diff
@@ -0,0 +1,48 @@
+
+FybrikApplication yaml - information provided about the workload and the datasets
+Fybrik manager
+Data catalog
+Data Governance Policy Manager
+Config Policy Manager
+FybrikModules
+
+## Issues Addressed by Taxonomy
+
+The taxonomy addresses the following:
+
+* Redundancy: No need for the same structures and values to be hardcoded in multiple places, such as in the fybrik manager and in the plugins.
+* Validation: Validates structures and values passed between components.
+* Dynamic Updates: New terms and new values can be added, removed and updated dynamically. For example, one can add new enforcement actions, new connection types, new purposes, etc without needing to redeploy fybrik.
+
+## Taxonomy Contributors
+
+Different actors and components define the contents of different parts of the taxonomy.  The following table describes the taxonomy and which component most logically owns each part of it. 
+
+| Taxonomy         | Contributing Component       | Actor              | Example Values                |
+|------------------|------------------------------|--------------------|-------------------------------|
+| catalog          | Data Catalog                 | Data Steward       | data stores, formats, metadata|
+| application      | Policy Manager, Data Catalog | Governance Officer | roles, intents                |
+| module           | Modules                      | Module Developer   | capabilities, transforms      |
+
+If, for example, a Data Governance Officer writes a policy that limits the use of sensitive data for marketing, then the possible valid intents such as marketing would be defined by him in the Data Policy Manager.  These values must be added to fybrik's taxonomy, either manually or via an automated feed, so that fybrik can validate the intent provided in a FybrikApplication yaml when a user's workload requests data.
+
+As new capabilities, transforms, data types, and protocols are made available via FybrikModules fybrik's module taxonomy must be updated.  Once updated these capabilities are available for use by other components, such as the Data Catalog and Data Governance Policy manager should they choose to leverage them.
+
+Default taxonomies are provided by fybrik, and are meant as a starting point on which to expand.
+
+## Validation Points
+
+Fybrik validates the structures and values it receives from all external components.  
+
+Webhooks are used for validation of FybrikApplication.yaml and for FybrikModule.yaml if webhooks are enabled.  If not enabled, the validation is done in the FybrikApplication controller (manager) and the FybrikModule controller.
+
+If webhooks are enabled then invalid yamls are automatically rejected and no resource is created.  However, if webhooks are turned off the resource is created and an error is indicated in its status should taxonomy validation fail.
```

Comment:
Maybe this sentence should be rephrased.

Reply:
Addressed.  I hope it's clearer now.